module/VuFind/src/VuFind/Controller/CombinedController.php

```diff
@@ -339,21 +339,32 @@ protected function adjustQueryForSettings($settings, $searchType = null)
         // Override the search type:
         $query->type = $searchType;
 
-        // Always leave noresults active (useful for 0-hit searches).
```

Comment:
Despite this comment, it doesn't seem like noresults recommendations have any way to appear without this PR.

Reply:
I'm seeing norecommend settings in the EDS column of Villanova's Bento box when I search for gibberish:

https://library.villanova.edu/Find/Combined/Results?dfApplied=1&lookfor=zdarearsar&type=AllFields

Must be coming from somewhere! I don't have time to dig into the details today, but if you want me to find time for it later, let me know. Is it possible that AJAX-loaded columns are doing something different from non-AJAX-loaded columns? (I ask just because we load EDS by AJAX, and it's showing recommendations, so maybe that's a factor).

Reply:
Ok I think I found the problem.  In AbstractSearch->getSeachSetupCallback(), Any `recommendOverride` settings in combined.ini overrride all of the default recommendation modules from searches.ini / EDS.ini etc.  So setting even a top recommendation in combined.ini overrides the noresults recommendations from EDS.ini.  

I think that's actually the intended behavior, override being global to all recommendations?  If so then we do need something like the include_recommendations_noresults setting in combined.ini.

Reply:
Thanks for digging into this! It does indeed seem like there's a problem here that needs to be addressed. I suspect that different bits of development at different points in time led us to this situation, and it was not a conscious decision to suppress noresults recommendations when top/side overrides were present... but it does seem like the most straightforward solution would be to add a new include setting to address this. Let me know if you need any help on the details or implementation there. Maybe it's worth doing that as a separate PR and then circling back to this one after it is merged.

Reply:
(Or maybe it makes more sense to do it all here -- I don't have a strong preference one way or the other!)

Reply:
I decided it was easier to resolve separately.  #3264